src/DssSpell.t.sol

```diff
@@ -2042,10 +2038,10 @@ contract DssSpellTest is DSTest, DSMath {
         vat.move(address(this), address(0x0), vat.dai(address(this)));
     }
 
-    function checkUNIV2LPIntegration(
+    function checkUNIV3LPIntegration(
```

Comment:
Maybe I'd just call it `checkUNILPIntegration` as we might add more UNIV2 collaterals.

Reply:
done